src/test/java/com/github/hanleyt/JerseyExtensionTest.java

```diff
@@ -22,7 +23,7 @@
 import java.lang.reflect.Modifier;
 import java.net.URI;
 
-import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I much prefer the Truth or AssertJ style of assertions.
I think that they're more fluent and readable, and that they give better error messages when the assertion fails.

What was the trouble you had with the truth library? Was it just that the GWT dependency bothered you? or it was causing issues in your project? 

I notice you didn't change the gradle file, so the truth dependency is still imported. If you do need to get rid of it then you can remove the gradle dependency.

However, since writing this extension I've started to think that assertJ is actually a better assertion library, as it has much better plugin support in intellij, so I would be happy to switch to that if you were interested in making the change. The API is almost exactly the same, so would just be updating imports.

What do you think?

Reply:
> What was the trouble you had with the truth library? Was it just that the GWT dependency bothered you? or it was causing issues in your project?

So, when I pulled down the truth library from Maven Central, I found I was getting errors because  of the GWT dependency.  Rather than try and track down why that was an issue, I decided to just convert the assertions over.

> I notice you didn't change the gradle file, so the truth dependency is still imported. If you do need to get rid of it then you can remove the gradle dependency.

Right, I missed that.  Sorry.  As you can probably tell, I wasn't using Gradle.  The Jersey project where I wanted to utilize the extension was a basic Eclipse Dynamic Web project with Ivy for dependency management.  I didn't have any experience with Gradle, so I copied the source files into my project and modified them.  Once I had everything working and tested, I moved the source files back into my fork in preparation for the PR.  I modified the docs, but forgot that Gradle should be updated.

> What do you think?
If the project were larger and had more complex assertions, then I think a more advanced assertion mechanism might be warranted.  In this case however, the assertions are pretty basic and are well handled (and, I think, fairly clear no matter what assertion library is used) so I'm not sure there's much benefit to be gained by using another assertion package and there's certainly a cost to having the dependency.  You're already using JUnit5 so that dependency is a given.  Unless you really think another assertion library makes things clearer than the JUnit5 assertions or you plan the code to grow to be larger and more complex than it currently is (thus requiring more complex assertions that are better handled by another assertion library), I would be inclined to stick with vanilla JUnit5 assertions.

It's a tough call.  I normally don't hesitate to add a new library even if it only saves me a couple of lines of code however in this case, no lines of code are saved and I don't think it's really any more or less readable with the additional assertions. 

Anyhow, that's just my opinion.  Sorry you asked? :)

Reply:
Hey Rob,
Sure, I'm happy to merge as is. Thanks for your work on this!
I'd prefer to keep the nice assertions but its just a small thing, doesn't change the API, and its just a small repo anyways so easy to change back.
On a side note I'd highly recommend getting familiar with gradle, it makes things a lot easier!
Thanks!
Tomás